RecoTauTag/RecoTau/python/tools/runTauIdMVA.py

```diff
@@ -126,8 +130,8 @@ def loadMVA_WPs_run2_2017(self):
             )
 
     def runTauID(self):
-        self.process.rerunMvaIsolationTask = cms.Task()
-        self.process.rerunMvaIsolationSequence = cms.Sequence()
+        setattr(self.process,"rerunMvaIsolationTask"+self.postfix,cms.Task())
+        setattr(self.process,"rerunMvaIsolationSequence"+self.postfix,cms.Sequence())
```

Comment:
```suggestion
        setattr(self.process,"rerunMvaIsolationTask"+self.postfix,cms.Task())
        _rerunMvaIsolationTask = getattr(self.process,"rerunMvaIsolationTask"+self.postfix)
```
and replace `getattr(self.process,"rerunMvaIsolationTask"+self.postfix)` with `_rerunMvaIsolationTask` below
for much less verbose and also faster access